### DISCUSSION TITLE
May be we need a LoginOption for Include user avatar url ?

### DISCUSSION THREAD

**Discussion by hoatruongdev09** [lupidan/apple-signin-unity#117] [Ideas]
Well avatar is necessary for some case like friend list. I will appreciate it if you make it available soon. Anyway it's a good plugin for authenticate with apple in Unity.

**Comment by lupidan** [comment]
As of today, Apple does not provide such option.
[They only provide options to get Full Name and Email](https://developer.apple.com/documentation/authenticationservices/asauthorizationscope?language=objc).

**Reply by hoatruongdev09** [reply]
Wow it's okay
